### DISCUSSION TITLE
Looking for flying bot to diffrent version of Kayak

### DISCUSSION THREAD

**Discussion by rafx205** [community/community#43100] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello,

I looking to bot which be compare a all version of Kayak and find the best option to buy ticket.

Ex. I want to fly one way from KRK to BVA in 10.02.2023. Bot search this fly in all website on Kayak I mean ex. Kayak.ch, Kayak. com, Kayak.pl, Kayak.fr etc. Because are have diffrent price for the same flying in diffrent version of Kayak.

Enyone can help me?

**Comment by queenofcorgis** [comment]
Hi @rafx205,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Kayak. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
